README.md

```diff
@@ -71,6 +71,46 @@ testAgent.on('metrics', console.log)
 testAgent.on('process.memory', console.log)
 ```
 
+## Metric Tags
+
+The agent is able to use environment variables as metrics tags. 
+Define a list of environment variables to be used as metric tags:  
+
+```
+# add the values of env. variables USER, PWD as tags to metrics
+export MONITORING_TAGS_FROM_ENV="USER, PWD"
+```
+
+Define static tags in `key=value` format 
```

Comment:
In STA do we use = or :?
We should be consistent if possible.

Reply:
Ok.Changed.